apps/back-office/src/app/(dashboard)/settings/team-members/components/ServicesEdit.tsx

```
@@ -0,0 +1,94 @@
+                    name="services"
+                    type="checkbox"
+                    value={service.id}
+                  />
+                  <label
+                    className="text-sm text-gray-700 cursor-pointer"
+                    htmlFor={service.id}
+                  >
+                    {service.type}
+                  </label>
+                </div>
+              ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)